stories/BoardOutline.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { CadViewer } from "src/CadViewer"
+
+function CustomBoard({ boardOutline }) {
+  return (
+    <board width="10mm" height="10mm" outline={boardOutline}>
```

Comment:
I think if you specify an outline you shouldnt need width or height

Reply:
We made outline optional and the width and height are still required, maybe later we can make it so that you are required to provide either

Reply:
Makes sense, Can you create an issue for that on the props repo?

Reply:
@seveibar I'll add that as an issue because

Reply:
Haha i love it 😀

Reply:
@seveibar this repo uses builder, core doesn't support board outline yet

Reply:
Ooh yea we should get core working here! Another great issue